packages/lib/services/plugins/PluginService.ts

```diff
@@ -177,8 +180,11 @@ export default class PluginService extends BaseService {
 		return this.plugins_[id];
 	}
 
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-	public unserializePluginSettings(settings: any): PluginSettings {
+	public unserializePluginSettings(settings: SerializedPluginSettings): PluginSettings {
+		if (typeof settings === 'string') {
+			settings = JSON.parse(settings) as PluginSettings;
+		}
```

Comment:
```suggestion
	public unserializePluginSettings(settings: SerializedPluginSettings): PluginSettings {
```
The `if (typeof settings === 'string')` logic is unnecessary, because `settings` is never a string. This is the case even though `serializePluginSettings` always returns a `string`.